### PR TITLE
Seqwho

### LINKED ISSUE #1388: seqwho
**Tool Details**

- Name of tool: seqwho
- Tool homepage: https://git.biohpc.swmed.edu/s181649/seqwho
- Tool description: Type Sequence files for species and assay
- Data suitable for MultiQC plot(s): fastqc like plots
- Most interesting data for General Stats table: Inferred Sequence Type, Inferred Species

**Tool output:**
TSV file of stats
PDF file of figures

- Log filename pattern: 
- File contents:

### PR DESCRIPTION
**Pull request by vsmalladi**
- [x] This comment contains a description of changes (with reason)
- [x] `CHANGELOG.md` has been updated
- [x] There is example tool output for tools in the <https://github.com/ewels/MultiQC_TestData> repository or attached to this PR
- [x] Code is tested and works locally (including with `--lint` flag)
- [x] `docs/README.md` is updated with link to below
- [x] `docs/modulename.md` is created
- [x] Everything that can be represented with a plot instead of a table is a plot
- [x] Report sections have a description and help text (with `self.add_section`)
- [x] There aren't any huge tables with > 6 columns (explain reasoning if so)
- [x] Module does not do any significant computational work

Close #1388

### CODE REVIEW

**Comment by vsmalladi** [comment]
@ewels any ideas when this can be merged in?

**Comment by ewels** [comment]
I'm finally back to working 100% now, so soon hopefully! Hoping to find time to work through all of the open PRs in the next few weeks if all goes well.

File: multiqc/utils/search_patterns.yaml [hunk 1]
```diff
@@ -767,6 +767,8 @@ samtools/rmdup:
   shared: true
 sargasso:
   fn: "overall_filtering_summary.txt"
+seqwho:
+  fn: "SeqWho_call.json"
```
Comment (vladsavelyev) [inline multiqc/utils/search_patterns.yaml:771]: Looking at the SeqWho code, the output is written to stdout and thus can have any name. I wonder if we can detect a file based on its contents?

**Review by vladsavelyev — COMMENTED** [review]
Merged with main and clean up - the code looks good now! Thanks a lot for the contribution, @vsmalladi.

One question about the search pattern.

File: multiqc/utils/search_patterns.yaml (hunk 1 above)
Comment (vladsavelyev) [reply multiqc/utils/search_patterns.yaml:771]: Perhaps this would be unique enough:

```yaml
seqwho:
  contents: '  "Per Base Seq": ['
```

Comment (vsmalladi) [reply multiqc/utils/search_patterns.yaml:771]: ya that should work.

**Review by vladsavelyev — APPROVED** [review]

**Comment by vsmalladi** [comment]
Thanks for getting this in

**Review by ewels — APPROVED** [review]
LGTM - the only bit I don't like is the `"stacking": None,`. This means the plots scale badly, as if you have 100 samples there are now 400 bars which gets unreadable very fast. However, I understand that it's wrong to stack percentages, so I'm not sure I have a better solution.

We've had similar situations before and talked about using a Parallel Coordinates plot. This isn't supported by MultiQC currently (beyond a hacky version with a line plot), but we may be able to add it in the future with Plotly now. That could be something to think about if we want to come back to this in the future.

No better solution that I can think of for now, so happy to merge as-is. Thanks @vsmalladi !

**Pull request merged**